scripts/utils/TagNuGetClientGitRepostoryOnRelease.ps1

```diff
@@ -9,10 +9,13 @@ Uses the Personal Access Token of NuGetLurker to automate the tagging process.
 PersonalAccessToken of the NuGetLurker account
 
 .PARAMETER VsTargetBranch
-The VS Branch that the NuGet build is being inserted into. 
+The VS Branch that the NuGet build is being inserted into.
 
 .PARAMETER BuildOutputPath
 The output path for NuGet Build artifacts.
+
+.PARAMETER BuildInfoJsonFile
+Path to the buildInfo.json file that is published as artifact for every build.
```

Comment:
The path passed to this script needs to be a path on the filesystem (read via `Get-Content`). The fact that the build saves it as an artifact that gets downloaded is immaterial to this script. It's useful to know for anyone investigating the NuGet.Client build and release pipelines, but I feel like this comment makes it sound like this script is going to download it from the build artifact, rather than a "local" file.